pkg/git/auth.go

```diff
@@ -79,9 +81,20 @@ func (b *Basic) populate(gitURL string) string {
 	}
 
 	u.User = url.UserPassword(b.Username, b.Password)
+	b.populateGitStore(*u)
+
 	return u.String()
 }
 
+func (b *Basic) populateGitStore(url url.URL) {
+	url.Path = ""
+	content := []byte(url.String())
```

Comment:
Consider adding more logging around git creds store population for better debugging.
```suggestion
	content := []byte(url.String())
	logrus.Infof("Populating git credentials to local file store: %s", GitCredentials)
```